src/main/java/org/jboss/netty/handler/ipfilter/IpV4Subnet.java

```diff
@@ -142,13 +135,13 @@ private void setNetId(String netId) throws UnknownHostException {
      * @return the integer representation
      */
     private static int toInt(InetAddress inetAddress1) {
-        byte[] address = inetAddress1.getAddress();
-        int net = 0;
-        for (byte addres : address) {
-            net <<= 8;
-            net |= addres & BYTE_ADDRESS_MASK;
-        }
-        return net;
+        byte[] octets = inetAddress1.getAddress();
+        assert octets.length == 4;
+
+        return (octets[0] & 0xff) << 24 |
+                (octets[1] & 0xff) << 16 |
+                (octets[2] & 0xff) << 8 |
+                octets[3] & 0xff;
     }
 
```

Comment:
This is a copy & paste from `4.0` branch, I noticed it preferred direct access vs `for-each byte` which creates an iterator.